app/controllers/admin/merchants_controller.rb

```diff
@@ -17,23 +17,35 @@ def update
         redirect_to admin_merchant_path(@merchant)
     end
   end
-  
+
   def show
     @merchant = Merchant.find(params[:id])
   end
-  
+
   def edit
     @merchant = Merchant.find(params[:id])
   end
-  
+
+  def new
+    @merchant = Merchant.new
+  end
+
+  def create
+    merchant = Merchant.new(name: params[:merchant][:name], enabled: false)
```

Comment:
I would suggest utilizing the `merchant_params` here instead of directly going into the params